src/app/core/cases/case-history.model.ts

```diff
@@ -0,0 +1,23 @@
+import { CaseTab } from './case-tab.model';
+import { CaseViewEvent } from './case-view-event.model';
+import { Type } from 'class-transformer';
+
+export class CaseHistory {
+  case_id?: string;
+  case_type: {
+    id: string,
+    name: string,
+    description?: string,
+    jurisdiction: {
```

Comment:
can't we reuse our Jurisdiction type here?
```jurisdiction: Jurisdiction```
we could also see if automatic deserialising works:
```
@Type(() => Jurisdicition)
jurisdiction: Jurisdiction
```

Reply:
It didn't allow Type declaration inside the case type literal declaration so moved case history case type as a separate class and reuse Jurisdiction